### ISSUE TITLE
Details of Data Preparation

### ISSUE THREAD

**Issue by zhujingsong** [gotjd709/Hooknet#1]
I've read the wsi_processing method used in MRNet, it's a useful and wonderful tool !!!

Since MRNet uses 'same padding', it's alright to get patches using util_multi.py. 

But HookNet uses 'valid padding'. If I get 284x284 patches using util_multi.py, then only about 1/4*1/4 of central region is used for pixel-wise classification, the other region is wasted. I assume possibly this is not a good way to process the WSI for HookNet.

**Comment by gotjd709** [comment]
I agree with your point. And I added the patch generation code for valid padding of HookNet in wsi_processing method. 
Check [wsi_method](https://github.com/CODiPAI-CMC/wsi_processing/tree/main).

Thanks for your interest in the wsi_processing method.
